cli/commands/run/server/agent.controller.js

```diff
@@ -55,7 +60,12 @@ module.exports = class AgentController {
         const txEvent = this.createTransactionEventFromGrpcRequest(
           call.request
         );
-        findings.push(...(await this.handleTransaction(txEvent)));
+
+        const returnedFindings = await this.handleTransaction(txEvent);
```

Comment:
Trying to figure out how to calculate the amount of memory taken up by the array. I looked at the scanner code and it looks like it [limits the response to 50kB now](https://github.com/forta-network/forta-node/commit/bde90d90c3efb3dc99023041c976381911d6e9d8).

If we still want to do that here the only way I can think of is using a package like this one: https://www.npmjs.com/package/object-sizeof

Reply:
Updated